products/ionic.md

```diff
@@ -11,50 +11,66 @@ activeSupportColumn: true
 eolColumn: Maintenance
 releaseColumn: true
 releaseDateColumn: true
+extendedSupportColumn: true
 
 auto:
 -   git: https://github.com/ionic-team/ionic-framework.git
 
+# support(R) = releaseDate(R+1)
 releases:
--   releaseCycle: "6"
-    releaseDate: 2021-12-08
+-   releaseCycle: "7"
+    releaseDate: 2023-03-29
     support: true
     eol: false
+    extendedSupport: false
+    latest: "7.0.0"
+    latestReleaseDate: 2023-03-29
+
+-   releaseCycle: "6"
+    releaseDate: 2021-12-08
+    support: 2023-03-29
+    eol: 2024-03-29
+    extendedSupport: false
```

Comment:
Extended support should end on Mar 29, 2024 according to https://ionicframework.com/docs/reference/support.